sdk/ml/azure-ai-ml/tests/dsl/unittests/test_command_builder.py

```diff
@@ -98,6 +98,11 @@ def command_with_artifact_inputs(self):
     def test_command_function(self, test_command):
         assert isinstance(test_command, Command)
         assert test_command._source == "BUILDER"
+
+        # Test print and jupyter rendering for the builder object
+        print(test_command)
```

Comment:
is there a way to actually test the print? Maybe checking stdout or something like that? Otherwise, unless someone ran pytest with `-s` (streaming) option, they wouldn't see the output from this print